### PR TITLE
Urdu Text Detection

### PR DESCRIPTION
**Pull request by abhishak3**
# Pull Request for DL-Simplified 💡

## Issue Title : Urdu Text Detection

- **Info about the related issue (Aim of the project)** : Create DL model which will identify the texts written in Urdu.
- **Name:** Abhishek Pandey
- **GitHub ID:** abhishak3
- **Email ID:** abhishek.pandey.3237@gmail.com
- **Idenitfy yourself: ** SSOC participant

Closes: #26

### Describe the add-ons or changes you've made 📃

This model can be created effectively using r-CNN, faster r-CNN, EAST or YOLO models without the need for creating our own. But here I've created a custom model from scratch using tensorflow and keras for detecting Urdu Text in images and creating a bounding box around them.

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
I've tested accuracy and loss of model across different optimization functions. I've also verified the results by displaying the output samples and training samples.

## Checklist: ☑️

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — APPROVED** [review]
Keep doing the good work. You are one of the worthy contributors of this open source project repo.

Your PR is approved and ready to be merged :100:
@abhishak3

**Review by abhisheks008 — APPROVED** [review]
Keep doing the good work. You are one of the worthy contributors of this open source project repo.

Your PR is approved and ready to be merged :100:
@abhishak3

**Comment by abhisheks008** [comment]
Hey @abhishak3 have you filled up the Devfolio usernames collection form?

**Comment by abhishak3** [comment]
I've already informed you on Discord about it. Please check.
